crates/fuel-core/src/graphql_api/ports.rs

```diff
@@ -130,3 +149,38 @@ pub trait DatabaseChain {
 
     fn base_chain_height(&self) -> StorageResult<DaBlockHeight>;
 }
+
+pub trait FindTx {
```

Comment:
You don't need to split each method into its trait. One trait with methods will be enough.

Reply:
The same goes for other traits, we don't need to split them=) We did that for Database because we built the new functionality on top of it.

Reply:
Yeh I agree and ideally these should be named to indicate which adapter they will match to. So this should probably just be the `TxPoolPort`

Reply:
Ok, will merge all of the traits into one adapter then